manager/src/test/java/com/redhat/service/bridge/manager/RhoasServiceTest.java

```diff
@@ -44,32 +44,11 @@ void beforeEach() {
     }
 
     @Test
-    void testWithRhoasDisabled() {
-        when(rhoasClientMock.createTopicAndGrantAccess(any(), any(), any())).thenThrow(UNWANTED_ACCESS_EXCEPTION);
-        when(rhoasClientMock.deleteTopicAndRevokeAccess(any(), any(), any())).thenThrow(UNWANTED_ACCESS_EXCEPTION);
-
-        RhoasService testService = buildTestService(false);
-
-        assertThatExceptionOfType(InternalPlatformException.class)
-                .isThrownBy(() -> testService.createTopicAndGrantAccessFor(TEST_BRIDGE_TOPIC_NAME, RhoasTopicAccessType.CONSUMER_AND_PRODUCER))
-                .withMessage(RhoasServiceImpl.RHOAS_DISABLED_ERROR_MESSAGE);
-        assertThatExceptionOfType(InternalPlatformException.class)
-                .isThrownBy(() -> testService.deleteTopicAndRevokeAccessFor(TEST_BRIDGE_TOPIC_NAME, RhoasTopicAccessType.CONSUMER_AND_PRODUCER))
-                .withMessage(RhoasServiceImpl.RHOAS_DISABLED_ERROR_MESSAGE);
-        assertThatExceptionOfType(InternalPlatformException.class)
-                .isThrownBy(() -> testService.createTopicAndGrantAccessFor(TEST_PROCESSOR_TOPIC_NAME, RhoasTopicAccessType.PRODUCER))
-                .withMessage(RhoasServiceImpl.RHOAS_DISABLED_ERROR_MESSAGE);
-        assertThatExceptionOfType(InternalPlatformException.class)
-                .isThrownBy(() -> testService.deleteTopicAndRevokeAccessFor(TEST_PROCESSOR_TOPIC_NAME, RhoasTopicAccessType.PRODUCER))
-                .withMessage(RhoasServiceImpl.RHOAS_DISABLED_ERROR_MESSAGE);
-    }
-
-    @Test
-    void testWithRhoasEnabled() {
+    void testWithAllWorking() {
```

Comment:
maybe a better name for `testWithAllWorking`?

Reply:
Changed :+1: